src/UI/Seller/src/app/models/environment.types.ts

```diff
@@ -11,7 +11,6 @@ export interface EnvironmentConfig {
   translateBlobUrl: string
   blobStorageUrl: string
   orderCloudApiUrl: string
-  buyerConfigs: Record<string, BuyerConfig>
   superProductFieldsToMonitor: string[]
```

Comment:
Are we using `superProductFieldsToMonitor` either?

Reply:
No we are not. I'll remove and update this PR.